keyboards/crkbd/keymaps/colemak_luna/keymap.c

```diff
@@ -0,0 +1,405 @@
+        oled_off();
+       
+   
+    }
+}
+
+static void print_status_narrow(void) {
+   
+
+    /* Print current layer */
+    oled_write("LAYER", false);
+
+    oled_set_cursor(0, 6);
+
+    switch (get_highest_layer(layer_state)) {
+        case 0:
+            oled_write("Base ", false);
+            rgb_matrix_reload_from_eeprom();
+            break;
+        case 1:
+            oled_write("Game ", false);
+            rgb_matrix_mode_noeeprom(RGB_MATRIX_ALPHAS_MODS);
+                rgb_matrix_sethsv_noeeprom(HSV_TEAL);
+            break;
+        case 2:
+            oled_write("Game2", false);
+            rgb_matrix_mode_noeeprom(RGB_MATRIX_ALPHAS_MODS);
+            rgb_matrix_sethsv_noeeprom(HSV_PURPLE);
+            break;
+        case 3:
+            oled_write("Lower", false);
+            //rgb_matrix_mode_noeeprom(RGB_MATRIX_ALPHAS_MODS);
+            //rgb_matrix_sethsv_noeeprom(HSV_TEAL);
+            break;
+        case 4:
+            oled_write("Raise", false);
+            //rgb_matrix_mode_noeeprom(RGB_MATRIX_ALPHAS_MODS);
+            //rgb_matrix_sethsv_noeeprom(HSV_RED);
+            break;
+        case 5:
+            oled_write("Adj  ", false);
+            rgb_matrix_mode_noeeprom(RGB_MATRIX_ALPHAS_MODS);
+            rgb_matrix_sethsv_noeeprom(HSV_YELLOW);
+            break;
+        default:
+            oled_write("Undef", false);
+    }
+
+
+    /* KEYBOARD PET RENDER START */
+
+    render_luna(0, 13);
+
+    /* KEYBOARD PET RENDER END */
+}
+
+oled_rotation_t oled_init_user(oled_rotation_t rotation) { return OLED_ROTATION_270; }
+
+bool oled_task_user(void) {
+    /* KEYBOARD PET VARIABLES START */
+
+    current_wpm   = get_current_wpm();
+    led_usb_state = host_keyboard_led_state();
+
+    /* KEYBOARD PET VARIABLES END */
+
+    if (is_keyboard_master()) {
+        print_status_narrow();
+    } else {
+        print_logo_narrow();
+    }
+    return false;
+}
+
+#endif
+ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        /* KEYBOARD PET STATUS START */
+ 
+        case KC_LCTL:
+            if (record->event.pressed) {
+                isSneaking = true;
+            } else {
+                isSneaking = false;
+            }
+            break;
+        case KC_SPC:
+            if (record->event.pressed) {
+                isJumping = true;
+                showedJump = false;
+            } else {
+                isJumping = false;
+            }
+            break;
+        case KC_CAPS:
+            if (record->event.pressed) {
+                isBarking = true;
+            } else {
+                isBarking = false;
+            }
+            break;
+ 
+        /* KEYBOARD PET STATUS END */
+        case KC_BLAYER:
+            if (record->event.pressed) {
+            // Do something when pressed
+            layer_invert(0);
```

Comment:
Will always be enabled, in theory.
```suggestion
```

Reply:
I think I did it like that for some reason, but maybe I didn't implement that or something, right now it seems like it can be just removed and then the whole custom keycode can be just replaced with a toggle layer.